### PR TITLE
feat: add article whisper-3-nova-3-deepgram-enhancements

### PR DESCRIPTION
**Pull request by naomi-lgbt**

### CODE REVIEW

File: articles/en/community/whisper-3-nova-3-deepgram-enhancements-1306408209740730371.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Whisper 3 and Nova-3 on Deepgram: Enhancing Transcription
+
+## Whisper 3 and Nova-3 on Deepgram: Enhancing Transcription
+
+Our community has been curious about the potential inclusion of Whisper 3 in the Whisper Cloud services and the upcoming Nova-3 solutions. As of now, Deepgram has no immediate plans to integrate Whisper 3 into the infrastructure.
+
+### Transcription Quality Comparison
+
+Feedback suggests that Whisper 3 provides superior transcription results under challenging conditions, such as noisy environments with overlapping speakers. This has prompted interest in integrating Whisper 3 capabilities with Deepgram's renowned diarizer technology.
```
Comment (jpvajda) [inline articles/en/community/whisper-3-nova-3-deepgram-enhancements-1306408209740730371.md:9]: I'm not sure if this is true.

File: articles/en/community/whisper-3-nova-3-deepgram-enhancements-1306408209740730371.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Whisper 3 and Nova-3 on Deepgram: Enhancing Transcription
+
+## Whisper 3 and Nova-3 on Deepgram: Enhancing Transcription
+
+Our community has been curious about the potential inclusion of Whisper 3 in the Whisper Cloud services and the upcoming Nova-3 solutions. As of now, Deepgram has no immediate plans to integrate Whisper 3 into the infrastructure.
```
Comment (jpvajda) [inline articles/en/community/whisper-3-nova-3-deepgram-enhancements-1306408209740730371.md:5]: This feels like a piece of content we wouldn't post as it's speculating on our long term roadmap. I'd suggest we don't post threads like this which talk about when / if we might release things.

**Review by jpvajda — CHANGES_REQUESTED** [review]
see comments for suggestions. I think we should close this PR and not publish it all given it's content about our roadmap plans.

File: articles/en/community/whisper-3-nova-3-deepgram-enhancements-1306408209740730371.md (hunk 2 above)
Comment (naomi-lgbt) [reply articles/en/community/whisper-3-nova-3-deepgram-enhancements-1306408209740730371.md:5]: Ticketed https://app.asana.com/0/1207576726792593/1208782836187262/f

**Pull request closed without merging**